.claude/commands/generate-release-notes.md

```diff
@@ -0,0 +1,299 @@
+  need to ask about it again.
+
+When you present a list of commits to be verified by the user, use a format something like this:
+
+```
+Verifying Validators - Major Commits
+  1. feat: LittDB Snapshots in https://github.com/Layr-Labs/eigenda/pull/1657
+  2. feat!: validator state cache in https://github.com/Layr-Labs/eigenda/pull/1903
+
+❓ Do any of these need to be moved to a different category or have their importance level changed?
+```
+
+THIS IS EXCEPTIONALLY IMPORTANT. VERIFY EACH COMMIT.
+
+At the end, double check that there are no remaining commits with the "[UNVERIFIED]" tag. If there are, 
+you need to circle back to the user and verify them.
+
+# Release Notes Template
+
+Below is a rough template for the release notes. Release notes are always markdown files. Sometimes a section
+might be empty, and that's okay. If that happens, omit that section from the final output.
+
+Note that sometimes there may be some major features that deserve their own section.
+
+```markdown
+
+# ${CURRENT_RELEASE} - Release Notes
+
+- Commit: `${CURRENT_COMMIT}`
+- Prior Release: `${PRIOR_RELEASE}`
+- Prior Commit: `${PRIOR_COMMIT}`
+
+A sentence or two describing if this release is optional or mandatory for validators. If it's mandatory,
+include a short reason why.
+
+# Validators
+
+A list of commits in a bulleted list that are relevant to validators.
+
+## Major Changes
+
+Put the major changes here.
+
+## Minor Changes
+
+Put the minor changes here.
+
+# Disperser
+
+A list of commits in a bulleted list that are relevant to the disperser.
+
+## Major Changes
+
+Put the major changes here.
+
+## Minor Changes
+
+Put the minor changes here.
+
+# Data API
+
+A list of commits in a bulleted list that are relevant to the Data API.
+
+## Major Changes
+
+Put the major changes here.
+
+## Minor Changes
+
+Put the minor changes here.
+
+# Contracts
+
+A list of commits in a bulleted list that are relevant to the smart contracts.
+
+## Major Changes
+
+Put the major changes here.
+
+## Minor Changes
+
+Put the minor changes here.
+
+# Integrations
+
+A list of commits in a bulleted list that are relevant to integrations.
+
+## Major Changes
+
+Put the major changes here.
+
+## Minor Changes
+
+Put the minor changes here.
+
+# Other
+
+## Major Changes
+
+Miscellaneous commits that don't fit into the above categories.
+
+Put the major changes here.
+
+## Minor Changes
+
+Put the minor changes here.
+```
+
+Here is an example of how an entry for a commit should look:
+
+```markdown
+- `feat`: add 'litt prune' CLI tool by @cody-littley in [#1857](https://github.com/Layr-Labs/eigenda/pull/1857)
+```
+
+The important information to include is:
+
+- The general type of commit (feat, fix, chore, docs, refactor, test, etc.)
+- A short description of what the commit does
+- The author of the commit (if available)
+- A link to the pull request or commit (if available)
```

Comment:
Shouldn't it always be a link to a PR?

Reply:
```
- A link to the pull request or commit (if available). Always prefer a link to a pull request, since that
  always has more information. But if you can't find the PR (e.g. an admin has force merged something), go with
  the link to the commit.
```